packages/app-marketplace-admin/src/components/apps-browse/app-browse-table.tsx

```diff
@@ -0,0 +1,101 @@
+import { ReapitConnectSession } from '@reapit/connect-session'
+import {
+  Button,
+  Table,
+  TableCell,
+  TableHeader,
+  TableHeadersRow,
+  TableRow,
+  TableRowContainer,
+  Title,
+} from '@reapit/elements'
+import { AppsBrowseConfigEnum, AppsBrowseConfigItemInterface } from '@reapit/foundations-ts-definitions'
+import { GetActionNames, getActions } from '@reapit/utils-common'
+import { useReapitUpdate } from '@reapit/utils-react'
+import React, { FC } from 'react'
+import { colorSquare } from './app-browse.styles'
+import { reapitConnectBrowserSession } from '../../core/connect-session'
+import { shleemy } from 'shleemy'
+
+export const AppBrowseTable: FC<{
```

Comment:
I see you've re-used the AppMarket code but can you re-name the components to something that makes sense please? Also, probably needs a tests?